### PR TITLE
Adds rattestation documentation

### PR DESCRIPTION
**Pull request by glad-dev**
Currently, the documentation does not contain any information on the remote attestation. This PR adds said information.

### CODE REVIEW

File: _docs/operate/rattestation.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+# Add route to CML through C0
+$ ip route add 172.23.0.1 via 172.23.0.2
+```
+
+## Software TPM Setup
+
+As previously mentioned, the attestation requires a TMP module, which we will emulate using a software TMP.
+Detailed instructions are available [in the deploy/qemu section](../deploy/qemu#use-tpm-emulation).
+
+## QMEU
```
Comment (jwsbck) [inline _docs/operate/rattestation.md:41]: QEMU

File: _docs/operate/rattestation.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+{:.no_toc}
+
+Remote attestation validates the runtime integrity of a system by comparing the system's measured values against a predefined baseline.
+The system's TPM cryptographically signs these measurements to guarantee their validity.
+
+In the following, we detail how to attest GyroidOS running in a QEMU VM.
+
+## Network Configuration
+
+The attestation tool cannot connect to the Container Management Layer (CML) when using a QEMU bride.
```
Comment (jwsbck) [inline _docs/operate/rattestation.md:20]: ~~bride~~ bridge

File: _docs/operate/rattestation.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+---
+title: Remote attestation (rattestation)
```
Comment (jwsbck) [inline _docs/operate/rattestation.md:2]: i would remove the '(rattestation)' to not blow up the side bar menu.

File: _docs/operate/rattestation.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+The system's TPM cryptographically signs these measurements to guarantee their validity.
+
+In the following, we detail how to attest GyroidOS running in a QEMU VM.
+
+## Network Configuration
+
+The attestation tool cannot connect to the Container Management Layer (CML) when using a QEMU bride.
+Thus, we use a TAP network.
+
+```bash
```
Comment (jwsbck) [inline _docs/operate/rattestation.md:23]: jekll seems not to support the ```bash syntax. It works for me without 'bash'. perhaps we can find another way, how we can have syntax highlighting.
grafik

File: _docs/operate/rattestation.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+The following commands are run on the host, not the QEMU VM.
+
+1. Clone the [cml repository](https://github.com/gyroidos/cml/)
+2. Navigate to the `cml/rattestation` directory.
+3. Install the dependencies listed [in the README](https://github.com/glad-dev/cml/tree/kirkstone/rattestation#readme).
+4. Create the log directory with `mkdir /data/logs` or change `#define LOGFILE_DIR "/data/logs"` in `main.c`.
+5. Build the attestation tool with `make`.
+6. Extract the first certificate from the trust chain located at `ws-yocto/out-yocto/test_certificates/ssig_subca.cert` and save it as `yocto.pem`.
+7. Update the expected values for the PCRs in `rattestation.conf` if they are available.
+8. Run the attestation tool with `./rattestation 172.23.0.1`.
```
Comment (jwsbck) [inline _docs/operate/rattestation.md:89]: Perhaps we can have an example output of a successful attestation here at the end?

**Review by jwsbck — CHANGES_REQUESTED** [review]

File: _docs/operate/rattestation.md (hunk 1 above)
Comment (glad-dev) [reply _docs/operate/rattestation.md]: Fixed in [5141b1d](https://github.com/gyroidos/gyroidos.github.io/pull/40/commits/5141b1d07bcc62fb6c5e0cc38a5bc9831a14aca8)

File: _docs/operate/rattestation.md (hunk 2 above)
Comment (glad-dev) [reply _docs/operate/rattestation.md]: Fixed in [8044a49](https://github.com/gyroidos/gyroidos.github.io/pull/40/commits/8044a49356a27231ee50d15e8953d08043d188ff)

File: _docs/operate/rattestation.md (hunk 3 above)
Comment (glad-dev) [reply _docs/operate/rattestation.md]: Fixed in [87b8b95](https://github.com/gyroidos/gyroidos.github.io/pull/40/commits/87b8b95eefd401b6fcecdea0770056ff37991fd7)

File: _docs/operate/rattestation.md (hunk 4 above)
Comment (glad-dev) [reply _docs/operate/rattestation.md]: I tried to use code-ray for syntax highlighting but it didn't work. For now I removed the `bash` keyword in [2d423f7](https://github.com/glad-dev/gyroidos.github.io/commit/2d423f78b994d7c2b59179809466aca3cd6c85dc)

File: _docs/operate/rattestation.md (hunk 5 above)
Comment (glad-dev) [reply _docs/operate/rattestation.md:89]: Added in [c79d42c](https://github.com/gyroidos/gyroidos.github.io/pull/40/commits/c79d42c040f0c700fbf11ee9a5d61a45f74f3f83)

**Review by jwsbck — APPROVED** [review]
very nice

**Pull request merged**